### DISCUSSION TITLE
Results or output figures for example data (water-se-e2-a)?

### DISCUSSION THREAD

**Discussion by hu-lm** [deepmodeling/deepmd-kit#1623] [Q&A]
Hi all,

To make sure dpmd-kit is working correctly on my machine, I am trying to determine if the test output I have is correct for the water example data in the se-e2-a type. Is there any official output data/information or error figures that I can check? Thank you so much!

**Comment by njzjz** [comment]
You can try this tutorial https://tutorials.deepmodeling.org/en/latest/Tutorials/DeePMD-kit/learnDoc/Handson-Tutorial%28v2.0.3%29.html
